packages/app/src/PodDetailsSidePanel.tsx

```diff
@@ -18,6 +19,79 @@ import styles from '../styles/LogSidePanel.module.scss';
 const CHART_HEIGHT = 300;
 const defaultTimeRange = parseTimeQuery('Past 1h', false);
 
+const PodDetailsProperty = React.memo(
+  ({ label, value }: { label: string; value?: string }) => {
+    if (!value) return null;
+    return (
+      <div className="pe-4">
+        <Text size="xs" color="gray.6">
+          {label}
+        </Text>
+        <Text size="sm" color="gray.3">
+          {value}
+        </Text>
+      </div>
+    );
+  },
+);
+
+const PodDetails = ({
+  podName,
+  dateRange,
+}: {
+  podName: string;
+  dateRange: [Date, Date];
+}) => {
+  const { data } = api.useMultiSeriesChart({
```

Comment:
hrmm this might be a bit expensive to pull this data as this will cause us to scan all rows in the db over the time range. 

We can probably use `useLogBatch` with limit 1 and extra fields set to those properties, which will just scan for 1 row of data and return immediately.

Reply:
ohh good point, updated it